src/api/endpoint/api/publish.js

```diff
@@ -17,8 +17,11 @@ export default function(router: Router, auth: IAuth, storage: IStorageHandler, c
   const can = allow(auth);
 
   // publishing a package
-  router.put('/:package/:_rev?/:revision?', can('publish'),
-    media(mime.getType('json')), expectJson, function(req: $RequestExtend, res: $ResponseExtend, next: $NextFunctionVer) {
+  router.put('/:package/:_rev?/:revision?', can('publish'), media(mime.getType('json')), expectJson, function(
```

Comment:
This is just formatting, if there is no need has to remains the same.

Reply:
fixed all extra formatting changes